scheduler_error_mailer/tests/test_scheduler_error_mailer.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2023 Hunki Enterprises BV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from unittest.mock import patch
+
+from odoo.tests.common import TransactionCase
+
+from odoo.addons.scheduler_error_mailer.hooks import post_init_hook
+
+
+class TestSchedulerErrorMailer(TransactionCase):
```

Comment:
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-18.0
Use BaseCommon as base test class to reduce overhead from tracking and boost the test suite.

Reply:
Done, thanks!